config.yml.example

```diff
@@ -50,3 +50,7 @@ influx:
   org: piaxe
   bucket: piaxe
   timezone: Europe/Berlin
+
+
+i2c:
```

Comment:
maybe should be called with a little bit less generic name like `i2c_display` :thinking: